app/components/osc-webhook-triggers/osc-webhook-triggers.component.js

```diff
@@ -95,22 +96,15 @@
       checkDuplicates(trigger);
     };
 
-    var addEmptyWebhookTrigger = function() {
+    ctrl.addEmptyWebhookTrigger = function() {
```

Comment:
if you update to `ctrl.addEmptyWebhookTrigger` here, you'll have to update the `addEmptyWebhookTrigger()` calls below, ~113.